src/main/java/com/teragrep/pth_10/ast/DPLParserCatalystContext.java

```diff
@@ -483,6 +506,7 @@ public DPLParserCatalystContext clone() {
             ctx.setDs(inDs);
             ctx.setConfig(config);
             ctx.setAuditInformation(auditInformation);
+            ctx.setQueryName(queryName);
```

Comment:
Is the `queryName` guaranteed to have a value or can there be issues with it being potentially `null`?